assets/css/editor.scss

```
@@ -60,3 +60,14 @@
 	margin: 0;
 	padding: 0;
 }
+
+.theme-twentytwenty {
+	.wp-block {
+		.wc-block-grid__product-title,
+		.wc-block-active-filters__title,
+		.wc-block-attribute-filter__title,
```

We need to add the stock filter title class here too, otherwise it doesn't get the same font size as the other blocks.
```suggestion
		.wc-block-attribute-filter__title,
		.wc-block-stock-filter__title,
```

---

Good catch. Apparently, I copied over the code from the first iteration, before the stock filter had been included. I just added the missing CSS class.